src/libs/LocalePhoneNumber.js

```diff
@@ -0,0 +1,54 @@
+import lodashGet from 'lodash/get';
+import loadashTrim from 'lodash/trim';
+import loadashIncludes from 'lodash/includes';
+import loadashStartsWith from 'lodash/startsWith';
+import loadashPadStart from 'lodash/padStart';
```

Comment:
Ah actually one thing here, can you update the import names here? They should be `lodash` not `loadash`

Reply:
Done